src/controller/characterController.ts

```diff
@@ -0,0 +1,13 @@
+import { db } from '../model/characterModel';
+import { Character } from '../model/type';
+
```

Comment:
The database will we called from scrapeController. We don't have routes for characters, so a character controller is not needed. Delete this file 🍍